frontend/src/views/DashboardManager/EmptyProgression.vue

```diff
@@ -0,0 +1,42 @@
+<template>
+  <v-card outlined class="pa-4">
+    <v-row>
+      <v-col cols="12" sm="6" md="5" class="d-flex flex-column">
+        <div v-for="measure in keyMeasures" :key="measure.id" class="d-flex my-3">
+          <div class="text-body-1 font-weight-bold">
+            <v-icon class="mr-2" size="20" :color="measure.mdiIconColor">{{ measure.mdiIcon }}</v-icon>
+            {{ measure.shortTitle }}
+          </div>
+          <v-spacer></v-spacer>
+          <v-chip small>À COMPLÈTER</v-chip>
+        </div>
+      </v-col>
+      <v-col cols="12" sm="6" md="7" class="text-center">
+        <v-card color="primary lighten-5 fill-height rounded">
+          <v-card-text class="font-weight-bold">Vous êtes prêt.e à vous lancer !</v-card-text>
```

Comment:
I think the point median is discouraged so we will have to change this at some point, might be for a different PR though. Maybe "Quels sont mes objectifs ?" as a way of moving back to the idea that the diagnostic is to help us provide users with info about what they need to improve and how to do it

Reply:
Yeah true, we should also tell Arthur for when he's designing the copy

Reply:
Autre suggestion ici : https://github.com/betagouv/ma-cantine/pull/2644/commits/0a7ff85ef44dac33e7cd2d1f53c22cac43560937